lib/v2/stheadline/std/realtime.js

```diff
@@ -11,16 +11,40 @@ module.exports = async (ctx) => {
     const { data: response } = await got(url);
     const $ = cheerio.load(response);
 
-    const items = $(`${category === '即時' ? '.moreNews > .col-md-4' : ''} .media-body > .my-2 > a`)
-        .toArray()
-        .map((item) => {
-            item = $(item);
-            return {
-                title: item.attr('title'),
-                link: item.attr('href'),
-                guid: item.attr('href').substring(0, item.attr('href').lastIndexOf('/')),
+    const items = (
+        await (async () => {
+            const headers = {
+                'Accept-Language': 'en-US,en;q=0.5',
+                'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
```

Comment:
`form` [will have](https://github.com/sindresorhus/got/tree/v11.8.6#form) the correct headers from got.